From 313b4d181a81c4f7a6d3f0a6d65aa822e3178661 Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Fri, 21 Oct 2016 10:52:05 -0400
Subject: [PATCH 1353/1722] drm/amd/dal: Don't need to set stream_enc_count

We look at num_stream_encoder from res_cap now.

Change-Id: I7802a1691568905b311648475e747e9e05e18ad7
Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Acked-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/dal/dc/dce110/dce110_resource.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/dal/dc/dce110/dce110_resource.c b/drivers/gpu/drm/amd/dal/dc/dce110/dce110_resource.c
index 696e8e7..be67f04 100644
--- a/drivers/gpu/drm/amd/dal/dc/dce110/dce110_resource.c
+++ b/drivers/gpu/drm/amd/dal/dc/dce110/dce110_resource.c
@@ -1174,7 +1174,6 @@ static bool construct(
 		pool->base.underlay_pipe_index = 2;
 	}
 
-	pool->base.stream_enc_count = 3;
 	dc->public.caps.max_downscale_ratio = 150;
 	dc->public.caps.i2c_speed_in_khz = 100;
 
-- 
2.7.4

